Apps/Patient/src/Program.cs

```diff
@@ -63,6 +62,8 @@ public static async Task Main(string[] args)
 
             Utility.ConfigureTracing(services, logger, configuration);
 
+            ExceptionHandling.ConfigureProblemDetails(services, environment);
+
```

Comment:
Nit: this could be moved back to its original line for a commit that's less noisy.

Reply:
I will leave as is in case it is decided to use out of box dot net 7. If that were the case, it should bed placed at the end to ensure the proper problem details api writer is used.